Source/WebDriver/SessionHost.cpp

```diff
@@ -105,7 +105,7 @@ void SessionHost::dispatchMessage(const String& message)
     responseHandler(WTFMove(response));
 }
 
-#if !USE(GLIB)
+#if !USE(GLIB) && !USE(INSPECTOR_SOCKET_SERVER)
```

Comment:
I'm not familiar with other ports, but for example, wouldn't the iOS fall under that criteria?
I thought there were many build configurations that did not use either GLIB or INSPECTOR_SERVER_SOCKET.

Reply:
Apple doesn't use this WebDriver implementation.

Reply:
I found 4 *.cmake files in the Source/WebDriver folder, Platform(GTK|PlayStation|Win|WPE).cmake.
Is it correct to assume that these are the only 4 types of build configurations now?
If that is correct, I can revise the conditions.

Reply:
Apple doesn't use CMake. The number of .cmake files doesn't match with the number of build configurations.

Reply:
I see, then, I'll change the function `SessionHost::isRemoteBrowser()` to common code.
It will return false by default and true only for remote automation.